Makefile

```diff
@@ -12,84 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PKG := github.com/dragonflyoss/Dragonfly
-SUPERNODE_SOURCE_HOME="${curDir}/../../src/supernode"
-BUILD_IMAGE ?= golang:1.10.4
-GOARCH := $(shell go env GOARCH)
-GOOS := $(shell go env GOOS)
-BUILD := $(shell git rev-parse HEAD)
-BUILD_PATH := release/${GOOS}_${GOARCH}
-
-LDFLAGS_DFGET = -ldflags "-X ${PKG}/cmd/dfget/app.Build=${BUILD}"
-LDFLAGS_DFDAEMON = -ldflags "-X ${PKG}/cmd/dfdaemon/app.Build=${BUILD}"
-
-ifeq ($(GOOS),darwin)
-    BUILD_PATH := release
-endif
+# You can assign 1 to USE_DOCKER so that you can build components of Dragonfly
+# with docker.
+USE_DOCKER=0 # Default: build components in the local environment.
```

Comment:
How about making the env variable as the default value?

```
USE_DOCKER?=0
```

in `hack/build-client.sh`

```bash
USE_DOCKER=${USE_DOCKER:-"0"}
```

then, we can execute build commands:

```bash
# use docker
export USE_DOCKER=1
make build # or: make build USE_DOCKER=1
./hack/build-client.sh
./hack/build-client.sh dfget

# use local env
unset USE_DOCKER
make build # or: make build USE_DOCKER=0
./hack/build-client.sh
./hack/build-client.sh dfget
./hack/build-client.sh dfdaemon
```

Reply:
Done